src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -998,6 +998,7 @@ TxQ::apply(
             // transaction, if one does.
             TxQAccount::TxMap::const_iterator const prevIter =
                 txQAcct.getPrevTx(txSeqProx);
+            auto const nextSeqProx = nextQueuableSeqImpl(sleAccount, lock);
```

Comment:
Consider deferring the assignment. Eg. [commit](https://github.com/miguelportilla/rippled/commit/80150588718bd68d567c520e5f90e54bb0eb460c) here.

Reply:
Thanks for this code! As I looked it over, I realized that there's no need for the `nextSeqProx` variable at all. Here's why:
* At https://github.com/miguelportilla/rippled/commit/80150588718bd68d567c520e5f90e54bb0eb460c#diff-5872318276bb51e9a07456270eb57f6760f41902adbad461db66b51a70a176afR1031, we know that `txSeqProx` is a sequence.
* At https://github.com/miguelportilla/rippled/commit/80150588718bd68d567c520e5f90e54bb0eb460c#diff-5872318276bb51e9a07456270eb57f6760f41902adbad461db66b51a70a176afR1134, we only look at `nextSeqProx` if `txSeqProx` is a ticket (not a sequence). That means that `nextSeqProx` will *always* be unseated, and thus `nextQueueableSeqImpl` will always get called.

So what I can do is get rid of `nextSeqProx`, change the first check back to the original code, and call `nextQueueableSeqImpl` when bumping the account sequence.